Compare-NET-Objects/CompareLogic.cs

```diff
@@ -124,12 +126,12 @@ public class CompareLogic : ICompareLogic
         /// </summary>
         public ComparisonConfig Config
         {
-            get { return _config;}
+            get => _config;
             set
             {
-                _config = value; 
-                VerifyConfig verifyConfig = new VerifyConfig();
+                var verifyConfig = new VerifyConfig();
                 verifyConfig.Verify(value);
+                _config = value;

```

Comment:
We should set config after validation. Otherwise, the invalid value can be used.